static/src/js/components/SpatialSelection/SpatialSelection.js

```diff
@@ -305,13 +323,18 @@ class SpatialSelection extends Component {
 
   // Callback from EditControl, called when the layer is edited
   onEdited() {
-    const { drawnLayer, drawnLayerType } = this.state
+    const { drawnLayers } = this.state
+    const [firstLayer = {}] = drawnLayers
```

Comment:
find the correct layer in drawnLayers instead of using the first layer, incase they edit a shapefile layer